packages/block-library/src/more/edit.js

```diff
@@ -10,7 +10,7 @@ import { getDefaultBlockName, createBlock } from '@wordpress/blocks';
 const DEFAULT_TEXT = __( 'Read more' );
 
 export default function MoreEdit( {
-	attributes: { customText, noTeaser },
+	attributes: { customText = '', noTeaser },
```

Comment:
Should we add a default to the attribute definition? I think that's what a lot of blocks do

Reply:
Sometimes, attribute updates require deprecations. I think undefined values don't get serialized, while empty does.

I can double-check, but adding deprecation for a simple fix like this is probably not worth it. What do you think?

Reply:
@ellatrix, I was wrong. It looks like there's no need for deprecation, at least for the current case.